### PR TITLE
Europython 2016 slides

### PR DESCRIPTION
**Pull request by fbochu**

### CODE REVIEW

File: docs/talks/europython2016/abstract.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Grocker, a Python build chain for Docker
+========================================
+
+Biography
+---------
+
+I'm a web (mostly backend) developer since 2008 and a pythonista since 2012.
+I'm currently working at Polyconseil mainly on the information systems behind
```
Comment (kit-cat) [inline docs/talks/europython2016/abstract.rst]: , mainly

File: docs/talks/europython2016/abstract.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Autolib' (electric car sharing service in Paris) and member of Polyconseil
```
Comment (kit-cat) [inline docs/talks/europython2016/abstract.rst]: car-sharing service in Paris). I'm also a member of the Polyconseil

File: docs/talks/europython2016/abstract.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+devops team.
+
+
+Abstract (suggested size 1500 chars)
```
Comment (kit-cat) [inline docs/talks/europython2016/abstract.rst]: remove `(suggested size 1500 chars)` ?

File: docs/talks/europython2016/abstract.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+------------------------------------
+
+At Polyconseil, we build the Autolib' (an electric car sharing service in
+Paris) information system. This system is based on many applications
+developed using web technologies, Django and sharing many internally
+maintained libraries for business logic.
+
+Packaging is a difficult problem when you are using Python and deploying large
+Python projects is even more difficult. When deploying on a live and user
```
Comment (kit-cat) [inline docs/talks/europython2016/abstract.rst]: user-focused

File: docs/talks/europython2016/abstract.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+focused system like Autolib', you can not rely on Pip and external PyPI
+servers which can fail and are not reproducible. Before using Docker we
+deployed our application using Debian packaging... And it was a maintenance
+hell: it take hours to build all Debian packages for our dependencies and
```
Comment (kit-cat) [inline docs/talks/europython2016/abstract.rst]: takes

**Comment by kit-cat** [comment]
@fbochu: j'avais pas vu la notif :(

**Comment by fbochu** [comment]
On master with 497b008a8252fd825dd59735a465607a8862da6c

**Pull request closed without merging**